### PR TITLE
Use static inline function instead of statement expression

### PR DESCRIPTION
**Pull request by michaelforney**
Not sure if this needs to be inlined, but I made it `inline` to match the existing macro.

### CODE REVIEW

**Comment by pali** [comment]
@michaelforney can you please rebase this pull request on top of master branch? I fixed Travis CI testing in this (now merged) PR: https://github.com/dosfstools/dosfstools/pull/122

**Comment by michaelforney** [comment]
Done. Looks like it's passing now.

File: src/lfn.c [hunk 1]
```diff
@@ -155,6 +147,14 @@ static void copy_lfn_part(unsigned char *dst, LFN_ENT * lfn)
     memcpy(dst + 22, lfn->name11_12, 4);
 }
 
+/* Convert name part in 'lfn' from unicode to ASCII */
+static inline char *cnv_this_part(LFN_ENT *lfn)
+{
+	unsigned char __part_uni[CHARS_PER_LFN*2];
```
Comment (pali) [inline src/lfn.c]: One thing: prefix `__` is reserved for compiler, so it must not be used for user variables in functions.

Comment (michaelforney) [reply src/lfn.c]: Done. I also adjusted the whitespace style slightly to match the surrounding code.

**Pull request merged**